src/pool/srv_pool.c

```diff
@@ -4686,6 +4692,15 @@ evict_iter_cb(daos_handle_t ih, d_iov_t *key, d_iov_t *val, void *varg)
 		return -DER_IO;
 	}
 
+	/* If we specified a machine name as a filter check before we do the realloc */
+	if (arg->eia_machine) {
+		struct pool_hdl	*hdl = (struct pool_hdl *)val->iov_buf;
+
+		if(strncmp(arg->eia_machine, hdl->ph_machine, MAXHOSTNAMELEN ) != 0) {
```

Comment:
(style)  line over 80 characters
(style)  space prohibited before that close parenthesis ')'
(style)  space required before the open parenthesis '('

Reply:
Fixed